include/swift/AST/DeclContext.h

```diff
@@ -115,8 +113,6 @@ enum class DeclContextKind : unsigned {
 /// \see SerializedLocalDeclContext.
```

Comment:
This comment is out of date. Is DeclContextKind::SerializedLocal also dead now?

Reply:
Yeah looks like we can probably just rip out SerializedLocalDeclContext at this point, I'll do that in a follow-up

Reply:
https://github.com/apple/swift/pull/70983